p2p/peer.go

```diff
@@ -45,6 +50,35 @@ func (pc peerConn) ID() ID {
 	return PubKeyToID(pc.conn.(*tmconn.SecretConnection).RemotePubKey())
 }
 
+// Return the IP from the connection RemoteAddr
+func (pc peerConn) RemoteIP() net.IP {
+	if len(pc.ips) > 0 {
+		return pc.ips[0]
+	}
+
+	if pc.conn == nil || pc.conn.RemoteAddr().String() == "pipe" {
```

Comment:
is this only nil during test? we should add comment either way that this little section is just for tests.